storage/pruning/pruningStorer_test.go

```diff
@@ -374,6 +374,57 @@ func TestNewPruningStorer_OldDataHasToBeRemoved(t *testing.T) {
 	assert.True(t, strings.Contains(err.Error(), "not found"))
 }
 
+func TestPruningStorer_GetFromOldEpochsWithoutCacheSearchesOnlyOldEpochs(t *testing.T) {
+	t.Parallel()
+
+	args := getDefaultArgs()
+	ps, _ := pruning.NewPruningStorer(args)
+
+	testKey1 := []byte("key1")
+	testVal1 := []byte("value1")
+	testKey2 := []byte("key2")
+	testVal2 := []byte("value2")
+
+	err := ps.Put(testKey1, testVal1)
+	assert.Nil(t, err)
+
+	err = ps.ChangeEpochSimple(1)
+	assert.Nil(t, err)
+	ps.SetEpochForPutOperation(1)
+
+	err = ps.Put(testKey2, testVal2)
+	assert.Nil(t, err)
+	ps.ClearCache()
```

Comment:
why do we clear the cache if the method shouldn't suppose to put it, anyway?
I would replace this with an  assert that the method did not stored something in the cache.

Reply:
This test used Put(), not PutWithoutCache(), so that is why the ClearCache() was needed. Changed so that PutWithoutCache() is used now.